### DISCUSSION TITLE
Kerberos Updates

### DISCUSSION THREAD

**Discussion by BenAigan** [git-lfs/git-lfs#4479] [Q&A]
All,

Apologies, I am not a developer nor do I have much knowledge in GoLang

I am able to build git-lfs with no issues but am not able to get the kerberos functionality working in my environment, if I am correct Git-LFS is using gokrb5.v5, can / could / should this be updated to gokrb5.v7 as this may help with my kerberos issues.

Your thoughts?

**Comment by bk2204** [comment]
Hey,

Sure, we can do that.  I had been planning to do that for a while, so thanks for reminding me.  Hopefully I'll have a PR up soon.

**Reply by BenAigan** [reply]
Thank you, I have built the code and tested and whilst the error output is different however I am still unsuccessful in getting the Git LFS objects without following this recipe.

I have to first destroy kerberos tickets, renew kerberos tickets, executing a curl on BB and then and only then does the clone work with Git LFS objects.

**Reply by BenAigan** [reply]
Ultimately we are receiving an unsupported encryption type error.  We will see if we can change this. Thanks for your help.

**Reply by bk2204** [reply]
Sure, happy to help.  Let us know if there's anything we can do to help out.

**Reply by BenAigan** [reply]
FYI: The issue we were having was that our git binary was built with an older Kerberos lib.

git depends on curl libraries to perform Kerberos and that the Kerberos dependencies of these did not match.  Once we updated the git Kerberos libs to match curl Kerberos libs everything worked as expected.
